.editorconfig

```diff
@@ -0,0 +1,3 @@
+[*.{kt,kts}] indent_size=4
```

Comment:
Can you insert a new line before `indent_size=4`?

Reply:
Yeah, sorry about that. Corrected in a2f6183